cns/cniconflist/testdata/fixtures/azure-windows-swift-overlay.conflist

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I know that in Windows AKS, the CNI conflist is generated dynamically based on a few things:
- If the customer specified [`DisableOutboundNAT`](https://learn.microsoft.com/en-us/azure/aks/nat-gateway#manually-disable-outboundnat-for-windows), then a certain `ExceptionList` for an `EndpointPolicy` will not be present
    - In the conflist I commented down below, this block is what I'm talking about:
```
{
     "Name":  "EndpointPolicy",
     "Value":  {
                   "Type":  "OutBoundNAT",
                   "ExceptionList":  [
                                         "fddc:be24:d690:87d::/64"
                                     ]
               }
 },
```
There are other settings that are dynamic, based on a few customer inputs, so we'll need to take all that into account when generating these Windows conflists

Here is where the Conflists are configured (altered) in Windows AKS: [AgentBaker - azurecnifunc.ps1](https://github.com/Azure/AgentBaker/blob/master/staging/cse/windows/azurecnifunc.ps1#L28-L223)

I would also talk to @AbelHu about what all settings in a customer cluster, can change the Conflist (or read the script I linked, he gave it to me originally)